src/components/RuleBook/index.js

```diff
@@ -4,125 +4,179 @@ import PropTypes from "prop-types";
-          <RuleContent>
-            🙆‍♂️ 🙆‍♂️ 🙆 🙆
-            <WrapCard>
+    <>
+      <ModalDiv onClick={handleClick}>
+        <ModalName>게임 규칙</ModalName>
+        <WrapRule>
+          <Rule>
+            <RuleContent>
+              <div>
+                <span>🙆‍♂️</span>
+                <span>🙆‍♂️</span>
+                <span>🙆</span>
+                <span>🙆</span>
+              </div>
               <p>Y</p>
-            </WrapCard>
-            <p>
-              각 100억 <Isprofit name="profit">이익</Isprofit>
-            </p>
-          </RuleContent>
-        </Rule>
-        <Rule>
-          <RuleContent>
-            🤦‍♂️ 🤦‍♂️ 🤦 🤦
-            <WrapCard>
+              <p>
+                각 100억 <Isprofit name="profit">이익</Isprofit>
+              </p>
+            </RuleContent>
+          </Rule>
+          <Rule>
+            <RuleContent>
+              <div>
+                <span>🤦‍♂️</span>
+                <span>🤦‍♂️</span>
+                <span>🤦</span>
+                <span>🤦</span>
+              </div>
               <p>X</p>
-            </WrapCard>
-            <p>
-              각 100억 <Isprofit name="loss">손실</Isprofit>
-            </p>
-          </RuleContent>
-        </Rule>
-        <Rule>
-          <RuleContent>
-            🤦‍♂️ 🤦‍♂️ 🙆 🤦
-            <WrapCard>
-              <p>Y</p>
-              <p>Y</p>
-              <p>X</p>
-              <p>Y</p>
-            </WrapCard>
-            <p>
-              X : 300억 <Isprofit name="profit">이익</Isprofit>
-            </p>
-            <p>
-              Y : 100억 <Isprofit name="loss">손실</Isprofit>
-            </p>
-          </RuleContent>
-        </Rule>
-        <Rule>
-          <RuleContent>
-            🤦‍♂️ 🙆‍♂️ 🙆 🙆
-            <WrapCard>
-              <p>Y</p>
-              <p>X</p>
-              <p>X</p>
-              <p>X</p>
-            </WrapCard>
-            <p>
-              X : 100억 <Isprofit name="profit">이익</Isprofit>
-            </p>
-            <p>
-              Y : 300억 <Isprofit name="loss">손실</Isprofit>
-            </p>
-          </RuleContent>
-        </Rule>
-        <Rule>
-          <RuleContent>
-            🤦‍♂️ 🤦‍♂️ 🙆 🙆
-            <WrapCard>
-              <p>Y</p>
-              <p>Y</p>
-              <p>X</p>
-              <p>X</p>
-            </WrapCard>
-            <p>
-              X : 200억 <Isprofit name="profit">이익</Isprofit>
-            </p>
-            <p>
-              Y : 200억 <Isprofit name="loss">손실</Isprofit>
-            </p>
-          </RuleContent>
-        </Rule>
-      </WrapRule>
-    </ModalDiv>
+              <p>
+                각 100억 <Isprofit name="loss">손실</Isprofit>
+              </p>
+            </RuleContent>
+          </Rule>
+          <Rule>
+            <RuleContent>
+              <div>
+                <span>🤦‍♂️</span>
+                <span>🤦‍♂️</span>
+                <span>🙆</span>
+                <span>🤦</span>
+              </div>
```

Comment:
혹시 이부분을 보니 남자얼굴이 y 여자얼굴이x를 뜻하는 거라면 
const RuleData = { 🤦‍♂️ : x, 🙆:y} 로 두고 활용하시는게 더 좋을 거 같습니다.
```suggestion
              <div>
                <span>🤦‍♂️</span>
                <span>🤦‍♂️</span>
                <span>🙆</span>
                <span>🤦</span>
              </div>
```